pkg/restore/restore.go

```diff
@@ -379,6 +379,7 @@ type context struct {
 
 type resourceClientKey struct {
 	resource  schema.GroupResource
+	version   schema.GroupVersion
```

Comment:
I'd probably just change `resource` to be a `schema.GroupVersionResource` so we don't need an additional field that's partially redundant.